modules/smart-agent_nagios-status-check/conf/readme.yaml

```diff
@@ -51,5 +32,5 @@ notes: |
 
   While SignalFx does not provide `Unknown` severity this module uses the `Major` severity for unknown alerts.
 
-  The metric is named `nagios_state.state`, you need to add an `extraDimensions` to your monitor in order to be 
+  The metric is named `nagios.state`, you need to add an `extraDimensions` to your monitor in order to be 
```

Comment:
this is now false while monitor takes `service` as mandatory parameter

Reply:
Removed from the doc